### ISSUE TITLE
Letterbox image utils function

### ISSUE THREAD

**Issue by kartik4949** [keras-team/keras-cv#153]

**Comment by sebastian-sz** [comment]
I think the `tf.image` equivalent could be [tf.image.resize_with_pad](https://www.tensorflow.org/api_docs/python/tf/image/resize_with_pad)

> Resize with kept aspect ratio and zero pad to desired dimensions. 

If you mean the YoloV5 implementation, the difference is they pad with 128 instead of zeros.

**Comment by kartik4949** [comment]
@sebastian-sz hmm.. 
resize_with_pad will require additional user implementation of providing padding, to maintain the aspect ratio of image,
but this one won't require that, just pass desired image_size and image tensor,
output will be resized image with maintained aspect ratio

**Comment by kartik4949** [comment]
yeah this is indeed similar, just saw your reference api,
idk if this takes fill_value as well...

**Comment by sebastian-sz** [comment]
@kartik4949 no need to close, I think this was a good issue and good idea :) 

Maybe reopen and we can see what people from the Keras team think?

**Comment by LukeWood** [comment]
Closing due to lack of details, feel free to open a new issue and fill out the feature request template.
